src/components/content/Home.js

```diff
@@ -40,7 +40,11 @@ const Home = ({
 };
 
 const DefaultHomeWrapper = styled.div``;
-
+const SecondHomeWrapper = styled.div`
+  h1 {
+    font-size: 100px;
```

Comment:
You should change this size to a little bit lower

Reply:
i change it